ubuntu/autoinstall.yml

```diff
@@ -0,0 +1,59 @@
+#cloud-config
+autoinstall:
+  version: 1
+  early-commands:
+    # Configure system-wide DNS, use Google DNS as generic server:
+    - 'sed -i "s/^nameserver [0-9.]\+/nameserver 8.8.8.8/g" /etc/resolv.conf'
+  packages:
+    - 'ubuntu-desktop'
```

Comment:
@DaniilKl why this in server installation?

Reply:
IIRC, because `netboot.xyz` uses Ubuntu server installer, but at the end we want to have desktop version.

Reply:
Ok, this is not the case for all installations. Also, this file format leads to weird issues on 24.04.1 (discussed more in the matrix). It should be changed to user-data, meta-data, and vendor-data files. I will provide fixes for this PR.

Reply:
Ok, thanks.